README.md

```diff
@@ -26,15 +26,15 @@ jobs:
     - name: Build
       run: ./my-build-script.sh
     - name: Log Build Failure
-      uses: ./.github/actions/cw-build-status/
+      uses: ros-tooling/action-cloudwatch-metrics@0.0.1
       with:
         status: 'failure'
-      if: failure() && github.ref == 'refs/heads/master'
+      if: failure() && github.event == 'schedule'
```

Comment:
Can you add a comment here stating what is happening?
I think this line is checking for build failures for scheduled events

Reply:
It is.

Reply:
Cool, I'd still like a comment since this wasn't completely obvious to me.
Otherwise LGTM.